run_scripts/start_bag.il

```diff
@@ -1185,15 +1184,21 @@ procedure( create_path_seg_helper(cv lay p0 p1 width start_s end_s)
             diag_ext = width * sqrt(2) / 2
         )
 
-        bext = 0
-        eext = 0
-        when( start_s == "round"
-            bext = width / 2
+        bext = width/2
+        eext = width/2
+        if( start_s == "round" then
             start_s = "custom"
+        else
+            when( start_s == "truncate"
+                bext = 0
+            )
         )
-        when( end_s == "round"
-            eext = width / 2
+        if( end_s == "round" then
             end_s = "custom"
+        else
+            when( end_s == "truncate"
+                eext = 0
+            )
         )
```

Comment:
I'm not sure about the origin or reason behind these changes and thus they were removed in the next commit (04322db).